certify-service/src/main/java/io/mosip/certify/controller/SvgTemplateController.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.certify.controller;
+
+import io.mosip.certify.core.entity.SvgTemplate;
+import io.mosip.certify.core.exception.CertifyException;
+import io.mosip.certify.core.exception.TemplateException;
+import io.mosip.certify.core.spi.SvgTemplateService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.CacheControl;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.ZoneId;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+@RestController
+@RequestMapping("/public")
+public class SvgTemplateController {
```

Comment:
We'll rename this to PublicController when functionality to host DIDs, public-keys, context is added. Ideally I'd like to keep a class with the `/public` route and individual methods to serve `/svg-template` , `/did` & `/pub-key` contents.

Reply:
I can understand the thought process, but we might also have api later to added a new svg-template, then the convention of public controller will confuse. 
Also for did and pub-key can also be exposed as part of system-info, since have the /certificate already there.

So on the public controller, we need to more time to decide on the design, lets keep it as renderingTemplateController  with url also as rendering-template for now and move on and later we can relook at this.

Reply:
The `/public` prefix is more of a convention we've decided earlier such that the DevOps team can enable caching for all URLs which have the word `/public` in them.

Reply:
I think certify is not matured to make that call, right now.